### DISCUSSION TITLE
LED Wiring Question

### DISCUSSION THREAD

**Discussion by vaderag** [darthcloud/BlueRetro#72] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/17175988/134683680-81dde462-2297-4647-bd3f-045a192b387d.png)

Currently using without and LED but thinking I might add one.
If i'm reading the above correctly, then the LED gets wired between ESP32 port 1 and GND, with a resistor in line and IO17 spliced in after the resistor - is this correct?

I don't actually know what R36 64.9 means tho - what ohm / banding resistor is this?

**Comment by sskylar** [comment]
+1 also trying to figure this out

**Comment by darthcloud** [comment, marked as answer]
Its standard practice to omit resistor, capacitor & inductor base unit in schematics. Only kilo , mega is added if needed.

So in this case its 64.9 ohm, its just a LED so it really doesn't matter anything between 64.9 and 1k will do.

**Reply by LuPe2593** [reply]
So with the LED you wire the cathode to ground and the anode you connect to 3v3/5v pin with a 64.9 ohm resistor. Then you wire the IO17 above the resistor on the anode. Is this correct? Or am I missing a step? Which 3v3/5v pin would be best to use? From what I remember, most of them are used up for the port detection current trigger mirror
